VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,245 @@
+# Validators FAQ
+
+## General concepts
+
+### What is a validator?
+
+Validators are the pillars of the network. Their role is to run a full node and participate in consensus by broadcasting votes which contain cryptographic signatures signed by their private key. Validators relay themselves to commit new blocks in the blockchain and receive rewards in exchange for their work. They must also participate in governance by voting on proposals.
```

Comment:
The third sentence `Validators relay themselves ...` makes no sense to me. What do you want to express?

Reply:
Just took the formulation of our validators page. To me it means that validators take turn in proposing blocks.

Reply:
Reformulated